config.json

```diff
@@ -56,7 +56,8 @@
     "dot-dsl",
     "custom-set",
     "nth-prime",
-    "forth"
+    "forth",
+    "beer-song"
```

Comment:
Does that list determine the order in which people get the exercises when the call `exercism fetch elixir`? If so, it totally doesn't!